src/org/jitsi/impl/neomedia/transform/OriginalHeaderBlockTransformEngine.java

```diff
@@ -225,15 +225,15 @@ public PacketTransformer getRTCPTransformer()
      */
     private void addExtension(RawPacket pkt)
     {
-        byte[] extensionBytes = new byte[11];
-        extensionBytes[0] = pkt.getPayloadType();
+        RawPacket.HeaderExtension he = pkt.addExtension(extensionID, 11);
 
-        RTPUtils.writeShort(extensionBytes, 1, (short) pkt.getSequenceNumber());
-        RTPUtils.writeInt(extensionBytes, 3, (int) pkt.getTimestamp());
-        RTPUtils.writeInt(extensionBytes, 7, pkt.getSSRC());
+        byte[] buf = he.getBuffer();
+        int off = he.getOffset();
 
-
-        pkt.addExtension(extensionID, extensionBytes);
+        buf[off + 1] = pkt.getPayloadType();
```

Comment:
Because the first byte contains the ID/length fields, data follows. Will document.